FinniversKit/Sources/Components/Ribbon/RibbonViewRepresentable.swift

```diff
@@ -0,0 +1,18 @@
+import SwiftUI
+
+public struct RibbonViewRepresentable: UIViewRepresentable {
```

Comment:
Looks like you decided to choose UIViewRepresentable because of using `overlay` inside SwiftUI, but why you need it?
Make things simple, just put it inside HStack with all other labels, but set property for your RibbonView 
`fixedSize` or `.layoutPriority(1)` (alternative of `.setContentHuggingPriority(.required, for: .horizontal)` in UIKit as far as I remember)